web/renderer/components/pageComponents/DatabasePage/ForRemotes/RemotesPage/RemoteRow.tsx

```diff
@@ -0,0 +1,41 @@
+import { RemoteFragment } from "@gen/graphql-types";
+import HideForNoWritesWrapper from "@components/util/HideForNoWritesWrapper";
+import { Button } from "@dolthub/react-components";
+import { FaRegTrashAlt } from "@react-icons/all-files/fa/FaRegTrashAlt";
+import { DatabaseParams } from "@lib/params";
+import css from "./index.module.css";
+
+type Props = {
+  remote: RemoteFragment;
+  onDeleteClicked: () => void;
+  params: DatabaseParams;
+};
+
+export default function RemoteRow({ remote, onDeleteClicked, params }: Props) {
+  return (
+    <tr>
+      <td>{remote.name}</td>
+      <td>{remote.url}</td>
+      <td>
+        {remote.fetchSpecs?.map((fs, i) => (
+          <span key={fs}>
+            {fs}
+            {i < (remote.fetchSpecs?.length ?? 0) - 1 ? ", " : ""}
+          </span>
+        ))}
```

Comment:
Can you just `remote.fetchSpecs?.join(",")` here? Does the span apply any styling?